ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala

```diff
@@ -303,6 +303,12 @@ object Cli {
         .action((dir, config) => config.copy(profileDir = Some(dir.toPath)))
         .text("Enable profiling and write the profiles into the given directory.")
 
+      opt[Boolean]("stack-traces")
+        .hidden()
+        .optional()
+        .action((enabled, config) => config.copy(stackTraces = enabled))
+        .text("Enable stack traces. Default is to enable them.")
```

Comment:
Is this text correct? It reads very odd.

Reply:
perhaps: `"Enable/Disable stack traces. Default is to enable them."`

Reply:
Good idea. Thanks.

Reply:
```suggestion
        .text("Enable/disable stack traces. Default is to enable them.")
```